### ISSUE TITLE
After installation the configuration of IMAP Accounts is not possible

### ISSUE THREAD

**Issue by fexxes** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#85]
Hello.

i am not sure if i am right here but i will try.

I installed a new Debian 9 System with php7.0 and apache2. I use the Debian Repro to install a fresh  tine20 installaiton. All setup checks shows up ok.

I use a mysql backend for my user and the creation of the initial user works well. After Login into the regular tine Frontend the only applications which shows up are Admin, Adressbook, Calender (comes up with an error) and Timetracker shows up in the menu. 

If i try to configure IMAP settings for my user the sektion is greyed out and i have no idea how to solve it. I check my old installation but the configuration seems the same. 

The tine.20.log shows to errors... 

`
2ba2a -- none -- - 2018-06-05T16:29:44+00:00 ERR (3): Tinebase_Exception::logExceptionToLogger::133 Zend_Db_Statement_Exception -> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tine20db.tine20_applications' doesn't exist, query was: SELECT `applications`.* FROM `tine20_applications` AS `applications`
2ba2a -- none -- - 2018-06-05T16:29:44+00:00 ERR (3): Tinebase_Exception::logExceptionToLogger::155 #0 .../vendor/zendframework/zendframework1/library/Zend/Db/Statement.php(303): Zend_Db_Statement_Pdo->_execute(Array)
#1 .../vendor/zendframework/zendframework1/library/Zend/Db/Adapter/Abstract.php(480): Zend_Db_Statement->execute(Array)
#2 .../vendor/zendframework/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('SELECT `applica...', Array)
#3 .../Tinebase/Backend/Sql/Abstract.php(795): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Select))
#4 .../Tinebase/Backend/Sql/Abstract.php(551): Tinebase_Backend_Sql_Abstract->_fetch(Object(Zend_Db_Select), 'fetch_all')
#5 .../Tinebase/Application.php(206): Tinebase_Backend_Sql_Abstract->search(NULL, NULL)
#6 .../Tinebase/Core.php(530): Tinebase_Application->getApplications()
#7 .../Tinebase/Core.php(506): Tinebase_Core::getPreCompiledContainer()
#8 .../Tinebase/Core.php(488): Tinebase_Core::setupContainer()
#9 .../Tinebase/Server/Expressive.php(86): Tinebase_Core::initFramework()
#10 .../Tinebase/Core.php(260): Tinebase_Server_Expressive->handle(Object(Zend\Http\PhpEnvironment\Request))
#11 .../index.php(24): Tinebase_Core::dispatchRequest()
#12 {main}
92207 -- none -- - 2018-06-05T16:29:58+00:00 ERR (3): Tinebase_Exception::logExceptionToLogger::133 Zend_Db_Statement_Exception -> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'tine20db.tine20_applications' doesn't exist, query was: SELECT `applications`.* FROM `tine20_applications` AS `applications`
92207 -- none -- - 2018-06-05T16:29:58+00:00 ERR (3): Tinebase_Exception::logExceptionToLogger::155 #0 .../vendor/zendframework/zendframework1/library/Zend/Db/Statement.php(303): Zend_Db_Statement_Pdo->_execute(Array)
#1 .../vendor/zendframework/zendframework1/library/Zend/Db/Adapter/Abstract.php(480): Zend_Db_Statement->execute(Array)
#2 .../vendor/zendframework/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('SELECT `applica...', Array)
#3 .../Tinebase/Backend/Sql/Abstract.php(795): Zend_Db_Adapter_Pdo_Abstract->query(Object(Zend_Db_Select))
#4 .../Tinebase/Backend/Sql/Abstract.php(551): Tinebase_Backend_Sql_Abstract->_fetch(Object(Zend_Db_Select), 'fetch_all')
#5 .../Tinebase/Application.php(206): Tinebase_Backend_Sql_Abstract->search(NULL, NULL)
#6 .../Tinebase/Core.php(530): Tinebase_Application->getApplications()
#7 .../Tinebase/Core.php(506): Tinebase_Core::getPreCompiledContainer()
#8 .../Tinebase/Core.php(488): Tinebase_Core::setupContainer()
#9 .../Tinebase/Server/Expressive.php(86): Tinebase_Core::initFramework()
#10 .../Tinebase/Core.php(260): Tinebase_Server_Expressive->handle(Object(Zend\Http\PhpEnvironment\Request))
#11 .../index.php(24): Tinebase_Core::dispatchRequest()
#12 {main}
`

An select on the table show that the error ist right. But there are other tables;

`MariaDB [tine20db]> select * from tine20_application;
ERROR 1146 (42S02): Table 'tine20db.tine20_application' doesn't exist
`

`MariaDB [tine20db]> show tables;
+-------------------------------------+
| Tables_in_tine20db                  |
+-------------------------------------+
| tine20_access_log                   |
| tine20_accounts                     |
| tine20_acsync_content               |
| tine20_acsync_device                |
| tine20_acsync_folder                |
| tine20_acsync_policy                |
| tine20_acsync_synckey               |
| tine20_adb_list_m_role              |
| tine20_addressbook                  |
| tine20_addressbook_image            |
| tine20_addressbook_industry         |
| tine20_addressbook_list_members     |
| tine20_addressbook_list_role        |
| tine20_addressbook_lists            |
| tine20_alarm                        |
| tine20_application_states           |
| tine20_application_tables           |
| tine20_applications                 |
| tine20_cal_attendee                 |
| tine20_cal_events                   |
| tine20_cal_exdate                   |
| tine20_cal_polls                    |
| tine20_cal_resources                |
| tine20_config                       |
| tine20_container                    |
| tine20_container_acl                |
| tine20_container_content            |
| tine20_credential_cache             |
| tine20_customfield                  |
| tine20_customfield_acl              |
| tine20_customfield_config           |
| tine20_departments                  |
| tine20_external_fulltext            |
| tine20_felamimail_account           |
| tine20_felamimail_cache_message     |
| tine20_felamimail_cache_message_bcc |
| tine20_felamimail_cache_message_cc  |
| tine20_felamimail_cache_message_to  |
| tine20_felamimail_cache_msg_flag    |
| tine20_felamimail_folder            |
| tine20_felamimail_sieve_rule        |
| tine20_felamimail_sieve_scriptpart  |
| tine20_felamimail_sieve_vacation    |
| tine20_filter                       |
| tine20_filter_acl                   |
| tine20_group_members                |
| tine20_groups                       |
| tine20_import                       |
| tine20_importexport_definition      |
| tine20_mailfiler_downloadlink       |
| tine20_mailfiler_message            |
| tine20_mailfiler_message_bcc        |
| tine20_mailfiler_message_cc         |
| tine20_mailfiler_message_to         |
| tine20_mailfiler_msg_flag           |
| tine20_note_types                   |
| tine20_notes                        |
| tine20_numberable                   |
| tine20_openid_assoc                 |
| tine20_openid_sites                 |
| tine20_path                         |
| tine20_preferences                  |
| tine20_record_observer              |
| tine20_relations                    |
| tine20_role_accounts                |
| tine20_role_rights                  |
| tine20_roles                        |
| tine20_scheduler_task               |
| tine20_state                        |
| tine20_tagging                      |
| tine20_tags                         |
| tine20_tags_acl                     |
| tine20_tags_context                 |
| tine20_temp_files                   |
| tine20_timemachine_modlog           |
| tine20_timetracker_timeaccount      |
| tine20_timetracker_timeaccount_fav  |
| tine20_timetracker_timesheet        |
| tine20_tree_fileobjects             |
| tine20_tree_filerevisions           |
| tine20_tree_node_acl                |
| tine20_tree_nodes                   |
+-------------------------------------+
82 rows in set (0.00 sec)
`

Maybe somebody has a hint what i have done wrong.

Grettings Fex

**Comment by hotmaillogin** [comment]
This is really interesting information for me. Thanks for sharing!  
-[url=http://hotmailwiki.com/hotmail-login]hotmail login[/url]

**Comment by pschuele** [comment]
you should install version 2018.02.6 at first - that should fix the errors in the tine20 log. and maybe the imap config is working afterwards, too!

**Comment by fexxes** [comment]
Hmm but i Installed Tine from the Debian Repo... so i have to delete it to download a other Version after this the Deb Repo Version is maybe working? 

I delete Tine days ago and migrate to a other solution. I think this business vs. community thing is not working so well. Thank you for your time but i think you can close this issue.

**Comment by pschuele** [comment]
> Hmm but i Installed Tine from the Debian Repo... so i have to delete it to download a other Version after this the Deb Repo Version is maybe working?

no, you just run "apt update && apt install tine20" to upgrade to the current version - it's easy when using the deb packages.


> I delete Tine days ago and migrate to a other solution. I think this business vs. community thing is not working so well.

ok - it's your choice. but I'm not sure what the the problem with community/business edition is. maybe you could explain it some more?

**Comment by fexxes** [comment]
Ok.. its to late now for an upgrade but until last weekend it was the newest version i could get. 

Yes, sure i can.... 

I start with tine20 years ago. And i had the feeling, not very sure because its years since then, that all tine stuff was well documentet. Now it feels like a mess... 

First thing is.. you search for tine2.0 and one of the top links are
https://www.tine20.com/willkommen-community/

First thing what i saq was "DIE BUSINESS EDITION".... which tells me how much cooler the Business Edition would be..Ok .. on this site there are a Link to the github. Now i am on github. I find the wiki and think cool .. a germean version ... but its only looks like there where a german version but in reality there are only one page german... this is no real problem when the english docoumentation would be fine. But it feels a little bit unprofessional or at least unfinished.. 

Now to the english version... 

Server requirements...

You say you can use php Version 5.6 or 7.0 or higher... the problem now is that there is no dedicated php7.0-date or php7.0-ctype package because they are included in other packageges.. like ctype is included in php7.0-commen. This is only the debian way of things but since debian is one of the systems you build packages for it maybe would be greate to give a better help. 

Install Howto

You descirpe that you can download a tar.gz file or try to fetch the software from the repos. Maybe the repo version will come up with all requirements and will install them but maybe it woul be nice to have, as example for debian, a line "to install requirements use apt-get install php70-commen *** *** "

a little bit down the text you find "configuration file config.inc.php" and "Preparing the database". If you use the repo than the db is already created and the config file is written. Sure its possible to change it but there is not the smales hint of it in the documentation which the project providet here on github. 

After this the documentation falls also a little bit short. Maybe its only me and all Applications and different configuration options are self explaind. For me it was not. 

If you where a new project, it would not be the problem. But tine20 is years old and there had to be more than this. In the moment it is not and this make me think that the community version is not so supported as they could be. 

Maybe i am wrong and it was only the wrong time i need a fresh installation of tine.. the issue traker looks like that something happens before... i started issue #85 and now we are on #6856.

In the moment i use a other software which dont offers Active Sync.. this brings up other problems and maybe i will come back to tine one time... 

I hope you dont feel offended.. it is not my intention...

**Comment by pschuele** [comment]
thanks for your thoughts - i think you have some valid points.

maybe you just stepped in at an unfortunate moment:
* we just migrated most of our own infrastructure (forum, wiki, issue tracker) to github
* we hope that the community helps with the wiki - the old one was very outdated and we needed a fresh start. i think, it will get better soon
* the issue tracker was just migrated to github (on the past weekend) - that's why the issue numbers "exploded"

and tine20 is not 20 years old, we started in 2007 :wink: 

> In the moment i use a other software which dont offers Active Sync.. this brings up other problems and maybe i will come back to tine one time...

yep, maybe we'll see each other again :)

> I hope you dont feel offended.. it is not my intention...

no problem at all. we always appreciate constructive feedback.
